lib/index.js

```diff
@@ -35,19 +33,33 @@ class Oppsy extends Events.EventEmitter {
 
         Hoek.assert(interval <= 2147483647, 'interval must be less than 2147483648');
         const host = Os.hostname();
-        this._interval = setInterval(() => {
 
-            Items.parallel.execute(this._tasks, (error, results) => {
+        this._interval = setInterval(async () => {
 
-                if (error) {
-                    this.emit('error', error);
-                }
-                else {
-                    results.host = host;
-                    this.emit('ops', results);
+            const tasks = [];
```

Comment:
Seems kind of wasteful to iterate over the object and push into an array after each interval. Can we change something we so don't need to do that every time the interval elapses?

Reply:
The issue here is the task tests would need to be totally rewritten as they change _tasks to a different set of tasks. I will probably come back to this but it shouldn't be anymore wasteful than the original use of Items

Reply:
do you still want this changed? I can go back to using Items if it makes you feel better for this to be hidden away. Really don't want a bunch of test rewrites to block release.